src/utils/p5Utils.js

```diff
@@ -83,29 +83,48 @@ class P5Util {
     });
   };
 
-  /**
-   * convert raw bytes to blob object
-   * @param {number[] | Uint8ClampedArray | ArrayLike<number>} raws
+  // /**
+  //  * convert raw bytes to blob object
+  //  * @param {number[] | Uint8ClampedArray | ArrayLike<number>} raws
+  //  * @param {number} width
+  //  * @param {number} height
+  //  * @returns {Promise<Blob>}
+  //  */
+  // async rawToBlob(raws, width, height) {
```

Comment:
Since you made your own helper function below, best to leave the rawToBlob() function unmodified.